src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -1553,7 +1553,7 @@ sap.ui.define([
 		var sStrucDefUrl;
```

Comment:
please add following test cases:
- oResource is `undefined`
- oResource  is not a valid resource (given object has no `resourceType` property)

Reply:
done

Reply:
done